platform-service/pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "cryptography>=41.0.7",
     "unstract-flags @ file:///${PROJECT_ROOT}/../unstract/flags",
     "requests>=2.31.0",
-    "unstract-sdk~=0.54.0rc1",
+    "unstract-sdk~=0.54.0rc2",
```

Comment:
We will have to hold on here because of some breaking changes, This is taken care in https://github.com/Zipstack/unstract/pull/862
If there are no blockers, pls revert this.